fiat-web/config/fiat.yml

```diff
@@ -4,6 +4,11 @@ server:
 redis:
   connection: ${services.redis.connection:redis://localhost:6379}
 
+services:
+  connection:
+    retryBackoff:
+      maxSeconds: 5
```

Comment:
Should this live in the normal `okHttpClient` config block vs. introducing a new section?

```
okHttpClient:
  retryBackOff:
    maxSeconds: 5
```

Consistency might see this as `okHttpClient.retries.maxElapsedBackoffMs` as we've used milliseconds for other config values.

Otherwise LGTM.